docs/components/checkbox/index.md

```diff
@@ -0,0 +1,55 @@
+# Checkbox
+
+Provides a Toucan-styled [checkbox element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox). If you are building forms, you may be interested in the CheckboxField component instead.
+
+## Checked Value
+
+To set the `checked` attribute of the checkbox, provide `@value`.
+
+```hbs
+<Form::Controls::Checkbox @value={{true}} />
+<Form::Controls::Checkbox @value={{false}} />
+```
+
+## Indeterminate
+
+Checkboxes have the ability to be in the [indeterminate state](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes). This is accomplished with `@isIndeterminate`. It is up to the consumer to decide what will force the checkbox into the indeterminate state. The [data-checked](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-checked) attribute is set to either "true", "false", or "mixed" depending on the state (where "mixed" is set if `@isIndeterminate={{true}}`).
```

Comment:
It's a bit confusing to me that all of the `aria-checked` attribute values are mentioned here, in the "Indeterminate" section.  Can we mention only the `mixed` one here, and then mention `true` and `false` up in the "Checked Value" section?

Reply:
I like it! Updating now, thank you!